lib/angular/index.js

```diff
@@ -0,0 +1,11 @@
+var modrain = require('../modrain');
+
+
+//TODO: This should probably be a separate module.
+module.exports = function() {
+  angular.module('wfm.core.modrain', []).factory('modrain', function testMediatorService() {
```

Comment:
Best to rename this to something non wfm related. This framework should not have any ties to wfm.
Is  "redhat.core.modrain" ok ?